templates/win2k12r2-deprecated.tpl.yaml

```
@@ -43,13 +43,15 @@ metadata:
         }, {
           "name": "windows-virtio-bus",
           "path": "jsonpath::.spec.domain.devices.disks[*].disk.bus",
+          "valid": "jsonpath::.spec.domain.devices.disks[*].disk.bus",
```

```suggestion
          "valid": "jsonpath::.spec.domain.devices.disks[*].disk",
```

The validation should be applied even when the user forgets to set the bus.

---

Changed.